src/main/webapp/js/adminActivityLog.es6

```diff
@@ -42,20 +67,23 @@ function submitLocalTimeAjaxRequest(time, googleId, role, entry) {
             $(localTimeDisplay).html('Loading error, please retry');
         },
         success(data) {
-            setTimeout(() => {
-                if (data.isError) {
-                    $(localTimeDisplay).html('Loading error, please retry');
-                } else {
-                    $(link).parent().html(`${originalTime}<mark><br>${data.logLocalTime}</mark>`);
-                }
-
-                setStatusMessage(data.statusForAjax, StatusType.INFO);
-            }, 500);
+            if (data.isError) {
```

Comment:
@junhaoyap It's a legacy setTimeout from beyond git history. Removal of setTimeouts don't seem to change anything other than to make them slower.